File: examples-sdk/swift/037-basic-vilmodel-derive/main.swift
// 037-basic-vilmodel-derive — Swift SDK equivalent
// Compile: vil compile --from swift --input 037-basic-vilmodel-derive/main.swift --release

let server = VilServer(name: "insurance-claim-processing", port: 8080)
let claims = ServiceProcess(name: "claims")
claims.endpoint(method: "POST", path: "/claims/submit", handler: "submit_claim")
claims.endpoint(method: "GET", path: "/claims/sample", handler: "sample_claim")
server.service(claims)
server.compile()

File: examples-sdk/kotlin/037-basic-vilmodel-derive/main.kt
// 037-basic-vilmodel-derive — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 037-basic-vilmodel-derive/main.kt --release

fun main() {
    val server = VilServer("insurance-claim-processing", 8080)
    val claims = ServiceProcess("claims")
    claims.endpoint("POST", "/claims/submit", "submit_claim")
    claims.endpoint("GET", "/claims/sample", "sample_claim")
    server.service(claims)
    server.compile()
}
